api_views/books.py

```diff
@@ -77,4 +84,28 @@ def get_by_title(book_title):
                 }
                 return Response(json.dumps(responseObject), 200, mimetype="application/json")
             else:
-                return Response(error_message_helper("Book not found!"), 404, mimetype="application/json")
\ No newline at end of file
+                return Response(error_message_helper("Book not found!"), 404, mimetype="application/json")
+
+
+def delete_book(book_id):
+    req_data = request.json()
```

Comment:
request.json() is incorrect - should be request.get_json(). This will cause runtime errors.